starter-code/french-onion-soup-starter-code/index.html

```diff
@@ -2,42 +2,52 @@
 <html lang="en">
 <head>
   <title>French Onion Soup Recipe</title>
-  <!-- Google fonts gives us access to the Open Sans Font -->
-  <!-- Google fonts is also awesome! Make sure to check it out -->
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
 <body>
-  French Onion Soup
-
-  We have been trying French onion soup in restaurants for years and my family
-and friends agree none can compare to my recipe for taste and simplicity of
-preparation.
-
-Ingredients
-1/2 cup unsalted butter
-2 tablespoons olive oil
-4 cups sliced onions
-4 (10.5 ounce) cans beef broth
-2 tablespoons dry sherry (optional)
-1 teaspoon dried thyme
-salt and pepper to taste
-4 slices French bread
-4 slices provolone cheese
-2 slices Swiss cheese, diced
-1/4 cup grated Parmesan cheese
-
-Directions
-Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
-continually stir until tender and translucent. Do not brown the onions.
-
-Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30
-minutes.
-
-Heat the oven broiler.
-
-Ladle soup into oven safe serving bowls and place one slice of bread on top of each
-(bread may be broken into pieces if you prefer). Layer each slice of bread with a slice of
-provolone, 1/2 slice diced Swiss and 1 tablespoon Parmesan cheese. Place bowls on
-cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.
+<div class="french-onion-image">
+  <h1>French Onion Soup</h1>
+</div>
+<section>
+  <p class="anectdote">We have been trying French onion soup in restaurants for years and my family
+  and friends agree none can compare to my recipe for taste and simplicity of
+  preparation.
+</section>
+  <div class="recipe-info">
```

Comment:
same as the apple pie clone, for this background image. Since it is not really a background, it would easier to manipulate its position and size by setting it as an image